docs/extensions.md

```diff
@@ -55,6 +55,13 @@ extensions:
     packages:
         - strace
         - ltrace
+    # Optional additional repos
```

Comment:
Feels like it's worth making more clear here the "globalness". Maybe just

```suggestion
    # Optional additional repos. Note this simply enables the repo globally.
```

Hmm, but actually it really wouldn't be hard to implement this properly. It'd be more or less just changing `generate_treefile` to define `repo_packages`. So... maybe let's just do that?

Reply:
OK too to leave as a follow-up.

Reply:
For development packages... it'd require something like [this](https://github.com/coreos/rpm-ostree/blob/25360ac6b834ce607d38a92aabaa210523c399ec/src/daemon/rpmostreed-os-experimental.cxx#L228-L240) copied into [here](https://github.com/coreos/rpm-ostree/blob/25360ac6b834ce607d38a92aabaa210523c399ec/src/app/rpmostree-compose-builtin-tree.cxx#L1614-L1618), but we'd also need to extend the Rust API to allow access to the repo names. Definitely can punt on that one.

Reply:
> Hmm, but actually it really wouldn't be hard to implement this properly. It'd be more or less just changing generate_treefile to define repo_packages. 

Yeah, I think the fact we need modules to influence global state here weakens the need to make this `repo_packages`, though I agree it'd be slightly more elegant.